service/src/main/java/uk/gov/hmcts/reform/fpl/model/Orders.java

```diff
@@ -55,7 +60,8 @@ public boolean orderContainsEPO() {
     public boolean isC1Order() {
         return isNotEmpty(orderType) && (this.getOrderType().contains(CHILD_ASSESSMENT_ORDER)
                 || isSecureAccommodationOrder()
-                || isRefuseContactWithChildApplication());
+                || isRefuseContactWithChildApplication()
+                || this.getOrderType().contains(CHILD_RECOVERY_ORDER));
```

Comment:
Should we use the isChildRecoveryOrder() function here instead

Reply:
Yes, you are right.